src/Aliases.jl

```diff
@@ -45,6 +45,13 @@ Base.@deprecate_binding isuinf is_uinf
 Base.@deprecate_binding isundefined is_undefined
 Base.@deprecate_binding isunknown is_unknown
 Base.@deprecate_binding isupper_triangular is_upper_triangular
+Base.@deprecate_binding lll_ctx LLLContext
```

Comment:
can you put these to the bottom of the file, so that the deprecations here and from https://github.com/Nemocas/Nemo.jl/pull/1621 are right after another?

Reply:
- Is it okay to put things at the bottom of the file ? (Line 56 seems to imply to me that all the lines after it were added in the 0.33 -> 0.34, so I didn't want to expand that list)
 - In your PR, you have them as aliases instead of `deprecate_binding`s. Is that preferable, and should I change mine to aliases as well ?

Reply:
I added a comment in https://github.com/Nemocas/Nemo.jl/pull/1621 stating the version these were introduced.
I had aliases there as a try to rename things in a non-breaking release. But this only works reasonably for functions, but not for types. Thus I just changed them back to deprecations. (we're doing breaking releases anyway, so I don't care)